frontend/pages/datasets/_workspace/_dataset/index.vue

```diff
@@ -77,7 +77,7 @@ export default {
     }),
     breadcrumbs() {
       return [
-        { link: { path: "/datasets" }, name: "Datasets" },
+        { link: { path: "/datasets" }, name: "Home" },
```

Comment:
link setup should  be aligned, so we should use name instead of the path here:
```suggestion
        { link: { name: "datasets" }, name: "Home" },
```